tldw_Server_API/app/core/DB_Management/media_db/legacy_wrappers.py

```
@@ -0,0 +1,120 @@
+)
+from tldw_Server_API.app.core.DB_Management.media_db.runtime.validation import (
+    MediaDbLike,
+    require_media_database_like,
+)
+
+
+def get_document_version(
+    db_instance: MediaDbLike,
+    media_id: int,
+    version_number: int | None = None,
+    include_content: bool = True,
```

Ruff (0.15.6)

[warning] Boolean-typed positional argument in function definition

(FBT001)